runtime/doc/vim_diff.txt

```diff
@@ -473,8 +473,9 @@ TUI:
   - Note: If you want to detect when Nvim is running in a terminal, use
     `has('gui_running')` |has()| or see |nvim_list_uis()| for an example of
     how to inspect the UI channel.
-- "builtin_x" means one of the |builtin-terms| was chosen, because the expected
-  terminfo file was not found on the system.
+- Neovim might optionally be compiled with unibilium, in case the terminfo
```

Comment:
I can't make sense of this sentence in my head. Should it be "in which case" maybe?